testing/values.go

```diff
@@ -57,7 +57,8 @@ var (
 
 	TestAccAddress    = "cosmos17dtl0mjt3t77kpuhg2edqzjpszulwhgzuj9ljs"
 	TestCoin          = sdk.NewCoin(sdk.DefaultBondDenom, sdkmath.NewInt(100))
-	TestCoins         = sdk.NewCoins(TestCoin)
+	TestAnotherCoin   = sdk.NewCoin("bond", sdkmath.NewInt(100))
```

Comment:
Could we not use the `SecondaryTestCoin` that already exists here?

Reply:
Updated it @gjermundgaraba